<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PuNeEtDocASAP" name="providerAccount.HandlingFrames" tests="1" failures="0" timestamp="8 Jan 2016 04:42:48 GMT" time="42.466" errors="1">
  <testcase name="setup" time="42.466" classname="providerAccount.HandlingFrames">
    <error type="java.lang.AssertionError" message="expected [false] but found [Error communicating with the remote browser. It may have died.
Build info: version: &#039;2.45.0&#039;, revision: &#039;32a636c&#039;, time: &#039;2015-03-05 22:01:35&#039;
System info: host: &#039;PuNeEtDocASAP&#039;, ip: &#039;192.168.8.103&#039;, os.name: &#039;Windows 8&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.2&#039;, java.version: &#039;1.7.0_07&#039;
Driver info: driver.version: EventFiringWebDriver]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [Error communicating with the remote browser. It may have died.
Build info: version: '2.45.0', revision: '32a636c', time: '2015-03-05 22:01:35'
System info: host: 'PuNeEtDocASAP', ip: '192.168.8.103', os.name: 'Windows 8', os.arch: 'x86', os.version: '6.2', java.version: '1.7.0_07'
Driver info: driver.version: EventFiringWebDriver]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at baseClass.BaseClass.getValue(BaseClass.java:80)
	at providerLogin.ProviderLoginPage.<init>(ProviderLoginPage.java:13)
	at providerAccount.HandlingFrames.setup(HandlingFrames.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- setup -->
</testsuite> <!-- providerAccount.HandlingFrames -->
